app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_comments_posts_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ ] [OPTIONAL] Please use the callback `after_save` to call the `update_users_comments_counter`, so the method is called automatically every time a new post has been saved to the database, then make the `update_users_comments_counter` method private.